### PR TITLE
Update Chef dependency to >= 16

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: kitchen.dokken.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+platforms:
+  - name: centos-8
+    driver:
+      image: dokken/centos-8
+      pid_one_command: /usr/lib/systemd/systemd
+
+  - name: centos-7
+    driver:
+      image: dokken/centos-7
+      pid_one_command: /usr/lib/systemd/systemd
```
Comment (ramereth) [inline kitchen.dokken.yml:25]: Let's not add this yet.

**Review by ramereth — CHANGES_REQUESTED** [review]

File: kitchen.dokken.yml (hunk 1 above)
Comment (TheREK3R) [reply kitchen.dokken.yml:25]: sorry, accident

Comment (TheREK3R) [reply kitchen.dokken.yml:25]: i automatically add this to cookbooks if im in dokken mode but try not to commit

**Comment by ramereth** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v1.6.0.  Have a nice day!

**Pull request merged**